client/hotrod-client/src/main/java/org/infinispan/client/hotrod/jmx/RemoteCacheClientStatisticsMXBean.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.client.hotrod.jmx;
+
+public interface RemoteCacheClientStatisticsMXBean {
+
+   /**
+    * Returns the number of remote cache gets
```

Comment:
If put() returns a value does it count as a get? (It should since we do count gets for `replace` but the code does not do that.)